src/zzz_od/operation/enter_game/stop_and_switch_account.py

```
@@ -0,0 +1,59 @@
+                log.warning('关闭游戏失败，2秒后重试一次')
+                time.sleep(2)
+                # 第二次尝试
+                res = self.ctx.controller.close_game()
+            # 轮询等待窗口释放，最多15秒
+            wait_left = 15
+            while res == 0 and wait_left > 0:
+                if not self.ctx.controller.is_game_window_ready:
+                    break
+                time.sleep(1)
+                wait_left -= 1
+            # 仍未关闭则返回重试，交给节点重试机制
```

Ruff (0.12.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)